app/assets/javascripts/modules/thumb_slider.js

```diff
@@ -10,14 +10,15 @@
 
     var toggleControl =
       jQuery(
-        '<div class="sul-i-navigation-show-more-1 sul-embed-thumb-slider-open-close"' +
+        '<button class="sul-i-navigation-show-more-1 sul-embed-thumb-slider-open-close"' +
           ' aria-expanded="true" aria-label="toggle thumbnail viewer">' +
-        '</div>'
+        '</button>'
       );
 
     var thumbList =
       jQuery(
-        '<div class="sul-embed-thumb-slider" style="display:none">' +
+        '<div class="sul-embed-thumb-slider" style="display:none"' +
+          ' aria-expanded="true">' +
```

Comment:
Since the default state of the slider is closed should the `aria-expanded` default to `false` (and then we'll toggle it to `true` once it is opened)?